man/adoc/bpftrace.adoc

```diff
@@ -2733,15 +2733,15 @@ Prints:
 === len
 
 .variants
-* `len(map m)`
+* `int64 len(map m)`
```

Comment:
signed or unsigned?

Reply:
signed according to semantic_analyser: https://github.com/bpftrace/bpftrace/blob/master/src/ast/passes/semantic_analyser.cpp#L1341